executor/aggregate_test.go

```diff
@@ -1288,9 +1283,17 @@ func (s *testSuiteAgg) TestIssue20658(c *C) {
 					}
 				}
 				c.Assert(ok, Equals, true)
-				tk.MustQuery(sql).Sort().Check(expected.Rows())
+				rows := tk.MustQuery(sql).Sort().Rows()
+
+				c.Assert(len(rows), Equals, len(expected))
+				for i := range rows {
+					v1, err := strconv.ParseFloat(rows[i][0].(string), 64)
+					c.Assert(err, IsNil)
+					v2, err := strconv.ParseFloat(expected[i][0].(string), 64)
+					c.Assert(err, IsNil)
+					c.Assert(math.Abs(v1-v2), Less, 0.005)
```

Comment:
Why is `0.005`? Instinctively, the error between float usually compares like `1e-3`.

Reply:
OK